src/test/denialofservice_tests.cpp

```diff
@@ -167,7 +168,7 @@ BOOST_FIXTURE_TEST_CASE(stale_tip_peer_management, OutboundTest)
         BOOST_CHECK(node->fDisconnect == false);
     }
 
-    SetMockTime(time_later);
+    elapse_time(delta);
```

Comment:
any reason for defining the `delta` so far away from first usage?
```suggestion
    const auto delta{3 * std::chrono::seconds{m_node.chainman->GetConsensus().nPowTargetSpacing} + 1s};
    elapse_time(delta);
```